styles/styles.css

```diff
@@ -128,30 +142,6 @@ h6 {
   font-weight: var(--spectrum-title-weight-s);
 }
 
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
-  margin-top: 0.8em;
-  margin-bottom: 0.25em;
-  scroll-margin: 2.5rem;
-}
-
-p,
-dl,
-ol,
-ul,
-blockquote {
-  margin-top: 0.8em;
-  margin-bottom: 0.25em;
-}
```

Comment:
What's going on with these global removals of margins? It looks like we're relying on the default browser styles now for general article content that uses these elements; should we still specify them?

Reply:
These came with the repo, so they weren't our styles. I do not want the base boilerplate styles getting mixed in with our cascade; When we build the blog pages in a later ticket, we can document expectations then.